### PR TITLE
[PLAY-1436] Dark Mode Fix: Contact and DRI

### PR DESCRIPTION
**Pull request by anthonymig88**
[PLAY-1436](https://runway.powerhrg.com/backlog_items/PLAY-1436)

**What does this PR do?**
Text for the Contact kit was not swapping over to the light text color and staying as default darker grey. Fixed now in both Rails and React. Also the date range inline icons for the body text were not swapping over to the lighter white color in react only.

**Screenshots:**
Screenshot 2024-07-16 at 2 00 31 PM
Screenshot 2024-07-12 at 11 40 53 AM

**How to test?** Steps to confirm the desired behavior:
1. Go to Contact Kit --> Verify text color
2. Go to Date Range Inline --> Verify icon color on React only

#### Checklist:
- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_contact/_contact.tsx [hunk 1]
```diff
@@ -83,25 +87,27 @@ const Contact = (props: ContactProps): React.ReactElement => {
     >
       <Body
           className="pb_contact_kit"
-          color="light"
+          color={dark ? "light" : "default"}
```
Comment (carloslimasd) [inline playbook/app/pb_kits/playbook/pb_contact/_contact.tsx:90]: @anthonymig88, just to make sure. Is this right?
I am asking because "light" was the value before the dark prop. Now, if it is dark, it will be light; if it is not, it will default.

Comment (anthonymig88) [reply playbook/app/pb_kits/playbook/pb_contact/_contact.tsx:90]: Thanks for the review and for pointing that out @carloslimasd ! You’re right, and to clarify:

- If dark is true, the color will be set to "light"
- If dark is false, the color will default to "default"

The goal was to ensure that the icon's color remains visually appropriate and consistent with the overall theme (light or dark) of the component. By making this change, we can ensure that the icon color adjusts dynamically!

**Review by carloslimasd — APPROVED** [review]
GREAT work, @anthonymig88!

File: playbook/app/pb_kits/playbook/pb_bar_graph/docs/_bar_graph_custom.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
-The `customOptions` prop provides comprehensive access to additional [Highcharts options](https://api.highcharts.com/highcharts/) that are not explicitly defined as props.
- It's important to note that certain options may require specific script imports to function properly.
+Note: If you are having trouble getting any Highcharts options to work, please match the formatting of our [staticOptions](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_bar_graph/_bar_graph.tsx#L85-L141). For example, `yAxis` will need to be wrapped with square brackets.
+
+You may also need to override any of the [defaults](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_bar_graph/_bar_graph.tsx#L45-L73) in order to get that options to work.
```
Comment (nickamantia) [inline playbook/app/pb_kits/playbook/pb_bar_graph/docs/_bar_graph_custom.md:3]: Hey! im not sure why this changes keeps following around lol. you can remove this change from this PR. ill do some research on why this keeps happening!

Comment (anthonymig88) [reply playbook/app/pb_kits/playbook/pb_bar_graph/docs/_bar_graph_custom.md]: Updated! Thanks for looking into this. Please keep me posted on any findings!

File: playbook/app/pb_kits/playbook/pb_date_range_inline/_date_range_inline.tsx [hunk 3]
```diff
@@ -36,59 +36,36 @@ const dateTimeIso = (dateValue: Date) => {
...
+    endDate,
     htmlOptions = {},
+    icon = false,
+    size = "sm",
     startDate,
-    endDate,
-    className,
   } = props;
 
-  const iconContent = () => {
```
Comment (nidaqg) [inline playbook/app/pb_kits/playbook/pb_date_range_inline/_date_range_inline.tsx:50]: Hey @anthonymig88 quick question: Any particular reason we're removing the function here and moving this extraction back into the return? Extraction is always better for performance and speed!

cc: @nickamantia @jasperfurniss

Comment (anthonymig88) [reply playbook/app/pb_kits/playbook/pb_date_range_inline/_date_range_inline.tsx:50]: @nidaqg I made the change to simplify the code. I agree that extraction is generally better for performance yet in this specific case, I felt the difference is negligible from what I've been learning. If performance becomes a concern with my change, I'm open to revisiting the changes.

**Review by nidaqg — APPROVED** [review]
